flink-dist/src/main/flink-bin/bin/flink-daemon.sh

```diff
@@ -85,6 +85,29 @@ id=$([ -f "$pid" ] && echo $(wc -l < "$pid") || echo "0")
 FLINK_LOG_PREFIX="${FLINK_LOG_DIR}/flink-${FLINK_IDENT_STRING}-${DAEMON}-${id}-${HOSTNAME}"
 log="${FLINK_LOG_PREFIX}.log"
 out="${FLINK_LOG_PREFIX}.out"
+gclog="${FLINK_LOG_PREFIX}.gc_log"
```

Comment:
Do you think we need to add PID in the gc_log filename?

Reply:
I prefer to keep the name the same with log & out files, if we determine to add PID in the gc_log filename, then we should also change the log&out file names.